### PR TITLE
(7.1.0 branch) HL7 message store related changes

### PR DESCRIPTION
**Pull request by nilminiwso2**
Since the HL7 message store is not supported in the Micro Integrator, updating the docs accordingly.

### CODE REVIEW

File: en/micro-integrator/docs/overview/about-this-release-7.1.0.md [hunk 1]
```diff
@@ -305,7 +305,29 @@ The following features, which are available in ESB runtimes, are removed from th
...
+			-
+		</td>
+	</tr>
+	<tr>
+		<td>
+			HL7 Console
+		</td>
+		<td>
+			The HL7 Console, which was available in the management console of previous ESBs, is removed in EI 7.x as it is not widely used.
+		</td>
```
Comment (Nirothipan) [inline en/micro-integrator/docs/overview/about-this-release-7.1.0.md:320]: I'm wondering whether we should explicitly mention this, as HL7 Message Store removal will imply this

**Review by Nirothipan — APPROVED** [review]

File: en/micro-integrator/docs/overview/about-this-release-7.1.0.md (hunk 1 above)
Comment (nilminiwso2) [reply en/micro-integrator/docs/overview/about-this-release-7.1.0.md:320]: I thought the HL7 console might have more use cases than the message store.

Comment (Nirothipan) [reply en/micro-integrator/docs/overview/about-this-release-7.1.0.md:320]: not really, its coupled with store only afaik

Comment (nilminiwso2) [reply en/micro-integrator/docs/overview/about-this-release-7.1.0.md]: updated as per suggestion

**Pull request merged**